packages/jest-cli/src/TestRunner.js

```diff
@@ -238,10 +247,22 @@ class TestRunner {
         aggregatedResults.success =
           aggregatedResults.numFailedTests === 0 &&
           aggregatedResults.numRuntimeErrorTestSuites === 0;
-        if (reporter.onRunComplete) {
-          reporter.onRunComplete(config, aggregatedResults);
-        }
-        return aggregatedResults;
+
+        return this._hasteMap.matchFiles(/\.snap/).then(res => {
```

Comment:
I see a problem here as we need to pass the aggregatedResults to add the count of the files we removed, other than that I think I can safely move it there.